src/main/java/meteordevelopment/meteorclient/utils/ReflectInit.java

```diff
@@ -59,6 +65,10 @@ private static <T extends Annotation> void reflectInit(Method task, Class<T> ann
             task.invoke(null);
         } catch (IllegalAccessException | InvocationTargetException e) {
             e.printStackTrace();
+        } catch (NullPointerException e) {
+            RuntimeException exception = new IllegalStateException("Method \"%s\" using Init annotations from non-static context".formatted(task.getName()));
```

Comment:
Wouldn't the catch clause also throw another null pointer exception because it is accessing the task?

Reply:
NPE comes from invoking a non-static method with a null instance, not from the method itself being null

This basically just improves the error message when someone uses PreInit or PostInit but without the static keyword

Reply:
Oh yeah, sorry missed that